data/nabadeeptalukdar.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Nabadeep Talukdar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I'm a frontend developer"
```

Add comma at the end 
```suggestion
  "bio": "I'm a frontend developer",
```